UnitsNet.Tests/GeneratedCode/VolumeTestsBase.g.cs

```diff
@@ -295,5 +295,291 @@ public void EqualsReturnsFalseOnNull()
             Volume cubicmeter = Volume.FromCubicMeters(1);
             Assert.IsFalse(cubicmeter.Equals(null));
         }
+
+        [Test]
+        public void ToStringReturnsCorrectNumberAndUnitWithDefaultUnit()
+        {
+            Volume.ToStringDefaultUnit = VolumeUnit.CubicMeter;
+            Volume cubicmeter = Volume.FromCubicMeters(1);
+            string cubicmeterString = cubicmeter.ToString();
+            Assert.AreEqual("1 " + UnitSystem.GetCached(null).GetDefaultAbbreviation(VolumeUnit.CubicMeter), cubicmeterString);
+        }
+
+        [Test]
+        public void ToStringReturnsCorrectNumberAndUnitWithCentiliterAsDefualtUnit()
```

Comment:
I like you went the whole 9 yards here, but I do think we should avoid adding tons of tests that basically test the same thing. I really just want to test the default behavior and one or two custom ToStringDefaultUnit values to ensure the implementation is working. Testing every unit seems to me to add little benefit since all units are generated the same way.

I propose you remove the powershell modification that generates these tests per unit, and instead manually write some test cases you find sufficient to verify the implementation.